src/MediaList.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.node,
+  childTag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  className: PropTypes.string,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  childTag: 'li',
+  tag: 'ul',
+};
+
+const MediaList = (props) => {
+  const {
+    children,
+    childTag,
+    className,
+    tag: Tag,
+    ...attributes,
+  } = props;
+
+  const classes = classNames(
+    className,
+    'media-list'
+  );
+
+  return (
+    <Tag {...attributes} className={classes}>
+      {React.Children.toArray(children).map((child, i) => {
+        return React.cloneElement(child, {
+          key: i,
```

Comment:
Not sure about `childTag`. Would it be better to push people to add `<Media tag="li"/>`. This seems a little strict by cloning each element inside of MediaList.